### PR TITLE
Adjust limits for CPU and memory for all apps.

### PR DESCRIPTION
**Pull request by Komzpa**
CPU limits are removed where possible.
Memory limits are bumped to 16Gi for smaller apps and 512Gi for potentially memory intensive ones.

### CODE REVIEW

**Comment by Komzpa** [comment]
@coderabbitai review

File: helm/osrm/values.yaml [hunk 1]
```diff
@@ -54,21 +54,19 @@ resources:
       memory: 256Mi
     limits:
       cpu: 1
```
Comment (dqunbp) [inline helm/osrm/values.yaml:56]: Skipped?

File: helm/raster-tiler/values.yaml [hunk 2]
```diff
@@ -30,7 +30,6 @@ resources:
     cpu: 2
```
Comment (dqunbp) [inline helm/raster-tiler/values.yaml:30]: Skipped?

**Review by dqunbp — DISMISSED** [review]

File: helm/raster-tiler/values.yaml (hunk 2 above)
Comment (Komzpa) [reply helm/raster-tiler/values.yaml:30]: it's requests, not limits

**Review by sofarsog00d — APPROVED** [review]

**Review by sofarsog00d — APPROVED** [review]

**Pull request merged**